lib/scaleway/bootstrap/master-cluster.tf

```diff
@@ -0,0 +1,41 @@
+resource "scaleway_k8s_cluster" "kubernetes_cluster" {
+  name    = "qovery-${var.kubernetes_cluster_id}"
+  version = "1.21.1"
+  cni     = "cilium"
+
+  tags    = ["qovery", "integration-test"] # TODO(benjaminch): put more usefull data in tags
+
+  autoscaler_config {
+    disable_scale_down              = false
+    scale_down_delay_after_add      = "5m"
+    estimator                       = "binpacking"
+    expander                        = "random"
+    ignore_daemonsets_utilization   = true
+    balance_similar_node_groups     = true
+    expendable_pods_priority_cutoff = -5
```

Comment:
I'm not sure about the value of this setting, we'll have to dig into it

Reply:
Removed it for the time being, to be investigated more in depth.